.github/workflows/ci.yml

```diff
@@ -48,3 +48,59 @@ jobs:
             PROJECT_NAME+="-${{matrix.compiler}}"
           fi
           rpm-gitoverlay --gitdir=gits build-overlay -s overlays/dnf5-unstable rpm --with "${{matrix.compiler}}" copr --owner "${{steps.setup-ci.outputs.copr-user}}" --project "$PROJECT_NAME" --chroots "$CHROOTS" --delete-project-after-days=7
+
+  copr-build:
```

Comment:
Now we are building packages in copr with gcc twice. Shouldn't we remove gcc from the `package-build` job and use it only for clang build checking?

Reply:
Yeah, I just thought I'd drop this in, see how it works and then remove the duplicity.

Reply:
I see. OK, let's what happens...